sonic_platform_base/sonic_sfp/sffbase.py

```diff
@@ -247,3 +250,27 @@ def dump_pretty(self, indict):
                     self.dec_indent()
             else:
                 print(self._indent, elem, ': ', elem_val)
+
+    @abc.abstractmethod
+    def read_eeprom_devid(self, port_num, devid, offset, num_bytes):
```

Comment:
@jleveque I think we should omit the port_num and let it come from sfp index ? 
Or 
Do we want the flexiblity of writing to other ports as well from a different port ?

Reply:
This is not the file for the abstract methods which the vendor implements. It is https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sfp_base.py

Reply:
fixed